src/main.ts

```diff
@@ -5,6 +5,7 @@ import run from "./run";
 import upload from "./upload";
 
 const VERSION = process.env.VERSION;
+const SKIP_UPLOAD = process.env.SKIP_UPLOAD;
```

Comment:
Prefixing this env variable name with codspeed would avoid some unexpected collisions with users env variables